README.md

```diff
@@ -4,21 +4,21 @@
 ## Table of Contents
 
 - [Best Email Marketing Tools for 2022](#best-email-marketing-tools-for-2022)
-- [Have ecommerce integration](#have-ecommerce-integration)
-- [Marketing automation](#marketing-automation)
-- [Tools](#tools)
+- [Marketing automation platform wich have ecommerce integration](#marketing-automation-platform-wich-have-ecommerce-integration)
+- [Automation Email Marketing platform](#automation-email-marketing-platform)
+- [Tools for effective works with e-mail](#tools-for-effective-works-with-e-mail)

```

Comment:
"Tools for effective work with emails", please update

Reply:
Done